opendut-edgar/src/service/start.rs

```diff
@@ -171,7 +174,68 @@ async fn handle_stream_message(
+            Message::ApplyPeerConfiguration(message) => match message {
+                ApplyPeerConfiguration { configuration: Some(configuration) } => {
+                    log::info!("Received configuration: {configuration:?}");
+                    match PeerConfiguration::try_from(configuration) {
+                        Err(error) => log::error!("Illegal PeerConfiguration: {error}"),
+                        Ok(configuration) => {
+                            for executor in configuration.executors.executors {
+                                match executor {
+                                    ExecutorDescriptor::Executable => log::warn!("Executing Executable not yet implemented."),
+                                    ExecutorDescriptor::Container {
+                                        engine,
+                                        name,
+                                        image,
+                                        volumes,
+                                        devices,
+                                        envs,
+                                        ports,
+                                        command,
+                                        args
+                                    } => {
+                                        let engine = match engine {
+                                            Engine::Docker => { "docker" }
+                                            Engine::Podman => { "podman" }
+                                        };
+                                        let mut cmd = Command::new(engine);
+                                        if let Some(command) = command {
+                                            cmd.arg(command.value());
+                                        } else {
+                                            cmd.arg("run");
+                                        }
+                                        if let Some(name) = name {
+                                            cmd.args(["--name", name.value()]);
+                                        }
+                                        for port in ports {
+                                            cmd.args(["-p", port.value()]);
+                                        }
+                                        for volume in volumes {
+                                            cmd.args(["--volume", volume.value()]);
+                                        }
+                                        for device in devices {
+                                            cmd.args(["--devices", device.value()]);
+                                        }
+                                        for env in envs {
+                                            cmd.arg(format!("-e {}={}", env.name, env.value));
+                                        }
+                                        for arg in args {
+                                            cmd.arg(arg.value());
```

Comment:
_"our"_ arguments have to be append after _"our"_ command. see my comment above.

synopsis: `podman run [options] image [command [arg …]]`